src/data/nat/prime.lean

```diff
@@ -112,6 +112,8 @@ theorem not_prime_one : ¬ prime 1 := by simp [prime]
 
 theorem prime_two : prime 2 := dec_trivial
 
+instance fact_prime_two : fact (nat.prime 2) := ⟨nat.prime_two⟩
```

Comment:
Based on @jcommelin's comment
```suggestion
local attribute [instance]
lemma fact_prime_two : fact (nat.prime 2) := ⟨nat.prime_two⟩
```

Reply:
I'll also write a library note and make the versions for 3/5/7